moped-editor/src/constants/projects.js

```diff
@@ -9,4 +9,5 @@ export const substantialCompletionDateTooltipText =
 export const agolFieldCharMax = {
   string: 2000,
   descriptionString: 10000,
+  projectNameFull: 1000,
```

Comment:
This isn't the best place to hang the this observation, but it's a place.

Check out how the check box, the X and the status pill shift down when the error is drawn. This is not a big deal, but I suspect it's not what you want to happen.

Pre-error:

![image](https://github.com/user-attachments/assets/6c3db412-a8ff-406c-9650-d04738df546e)


With error:

![image](https://github.com/user-attachments/assets/eaa6dfcb-4d5a-4dbd-aa93-3d27181d2fba)

Reply:
Thanks for catching this! I went ahead and aligned the buttons and badge to be centered in the flex row.